lib/backy.rb

```diff
@@ -1,20 +1,31 @@
 # frozen_string_literal: true
 
 require "active_support/core_ext/integer/time"
+require "active_support/core_ext/object/blank"
 
-require_relative "backy/app_config"
-require_relative "backy/db_config"
-require_relative "backy/base"
+require_relative "backy/configuration"
+require_relative "backy/db"
 require_relative "backy/list"
 require_relative "backy/pg_dump"
 require_relative "backy/pg_restore"
-require_relative "backy/s3_config"
+require_relative "backy/s3"
 require_relative "backy/s3_list"
 require_relative "backy/s3_load"
 require_relative "backy/s3_save"
 require_relative "backy/version"
 
+require_relative "backy/railtie" if defined?(Rails::Railtie)
+
 module Backy
   class Error < StandardError; end
-  # Your code goes here...
+
+  LOG_FILE = "log/db_backup.log"
```

Comment:
This should be configurable also. Maybe with options for logging to stdout?

Reply:
Good idea! Will fix